website/blog/2019-10-28-react-native-doctor.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Meet Doctor, a new React Native command
+author: Lucas Bento
+authorTitle: React Native Community
+authorURL: https://twitter.com/lbentosilva
+authorImageURL: https://avatars3.githubusercontent.com/u/6207220?s=460&v=4
+authorTwitter: lbentosilva
+category: announcements
+---
+
+After 160+ commits from 6 contributors, the React Native Community is excited to launch `doctor`. It's a brand-new React Native CLI command to help you out with getting started, troubleshooting and automatically fixing errors with your development environment. `doctor` is heavily inspired by [Expo](https://expo.io/) and [Homebrew](https://brew.sh/)'s own doctor command with a pinch of UI inspired by [Jest](https://jestjs.io/).
```

Comment:
To be fair, that number comes from squashed commits, wouldn't that number be correct if counted on top of all the PRs?

Reply:
Nah, I don't think it makes sense. 160 commits seems like a lot, but really to me only the squashed ones count, because that's the end shape of the PR and hence a commit in master

Reply:
Makes sense, gonna change to your suggestion later today.